pgroonga/migrations/0001_enable.py

```diff
@@ -17,21 +17,25 @@ class Migration(migrations.Migration):
 
 SET search_path = %(SCHEMA)s,public,pgroonga,pg_catalog;
 
-ALTER TABLE zerver_message ADD COLUMN search_pgroonga text;
+ALTER TABLE zerver_message ADD COLUMN search_pgroonga_topic text;
+ALTER TABLE zerver_message ADD COLUMN search_pgroonga_rendered_content text;
```

Comment:
Generally, we should not change existing migration files. Existing migration files aren't ran on already deployed environment.
We need to add a new migration file (`pgroonga/migrations/0004_XXX.py`) when we change schema.

Could you implement migration in a new migration file instead of changing existing migration files?

Reply:
Yea you're right, will fix this and push.